samples/stock/interactive-docs/series-type/demo.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/heikinashi.js"></script>
+<script src="https://code.highcharts.com/stock/modules/hollowcandlestick.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container" class="chart"></div>
+<div id="button-box">
+    <label for="select-series">Change series type:</label>
+    <select name="select-series" id="select-series">
+        <option value="ohlc">OHLC</option>
+        <option value="candlestick">Candlestick</option>
+        <option value="hollowcandlestick">Hollow candlestick</option>
+        <option value="heikinashi">Heikin Ashi</option>
```

Comment:
1. Missing HLC

2. How about adding all other popular series types? With separators, like this:

```html
<select>
    <option disabled>Stock specific series types:</option>
    < ... >
    <option disabled>A selection of basic series types:</option>
    <option>Spline</option>
    <option>Area</option>
    <option>Area Range</option>
    <option>Column</option>
    <option>Column Range</option>
</select>
```

Reply:
I agree with both suggestions, if we exclude `arearange` and `columnrange` (as they require different datasets to show how they work) then it looks fine 👍

Reply:
I would resign from Scatter, because without extra config it looks bad, though. It has different default tooltip and no DataGrouping.